src/common/dataHooks/useSearchLoader.js

```diff
@@ -56,8 +57,8 @@ function useSearchLoader({ searchParams }) {
       case TYPE_SONG:
       case TYPE_STORY:
         return {
+          ...storySummaryAdaptor({ item: result?.entry }),
           ...baseObject,
-          translations: [result?.entry?.title_translation],
```

Comment:
I'll do a new PR to get this working for songs, after Cara is done with the data hooks for songs (so I can reuse the data adaptor bits)

Reply:
Update: I just added it to this PR since Cara merged first :)